### PR TITLE
Fix #45: Add search box on experiment page

### LINKED ISSUE #45: Add search box on experiments page
Allow users to filter charts by title through a search box on the experiments page. For example, if they type **client_**, the following charts might appear:

client_crashes
client_daus
client_count
etc.

### PR DESCRIPTION
**Pull request by openjck**
The search box isn't beautiful yet, but I plan to address that in a future PR.

### CODE REVIEW

**Comment by openjck** [comment]
A lot of the changes in *Experiment.js* are just whitespace changes. This view will make it easier to read:

https://github.com/mozilla/firefox-test-tube/pull/72/files?w=1

File: src/components/views/Experiment.js [hunk 1]
```diff
@@ -102,8 +102,25 @@ export default class extends React.Component {
...
+                            <section id="experiment-ping-counts">
+                                <h5>Pings</h5>
+                                {this.getCountDL(this.props.populations, 'total_pings')}
+                            </section>
+                        </section>
+                        {maybeAuthors}
+                    </section>
                     <aside id="experiment-options">
                         <h3>Options</h3>
+                        <input type="text" placeholder="Search..." onKeyUp={this.props.onSearch} />
```
Comment (spasovski) [inline src/components/views/Experiment.js]: `type="search"` and it will default to text for older browsers....which will fail on CSS grid anyway 😁

Comment (openjck) [reply src/components/views/Experiment.js]: Oh man. I can't believe I didn't know about this. Updated!

**Comment by spasovski** [comment]
Looks and works great - feel free to merge as is. We might want to consider limiting filtering only once at least X characters are entered though I doubt this matters for our users.

**Comment by openjck** [comment]
I'll keep that in mind. Do you bring this up for the performance benefits or something else?

**Comment by spasovski** [comment]
Perf mostly since it will start filtering right away during typing (with 100ms debounce) though this used to be a bigger deal in the async fetch results days...here we're just narrowing down an existing array in memory. The more I think about it the more I realize it's unnecessary really since we should make that decision on user feedback (if any) in the future.

**Pull request merged**